model/main.mod

```diff
@@ -62,9 +76,9 @@ subject to cWorkHoursLimit{n in sNurses}:
 subject to cDailyShiftLimit{nurse in sNurses, day in sDays}:
     sum{shift in sShifts} vSchedule[nurse, day, shift] <= 1;
     
-# respect night shifts per week limit
+# max 6 night shifts per week
```

Comment:
I'd revert to "respect night shifts per week limit"

Reply:
Right